projects/coordinator/src/main/java/org/batfish/coordinator/PoolMgrService.java

```diff
@@ -36,6 +37,33 @@ public JSONArray getInfo() {
   }
 
   @GET
+  @Path(CoordConsts.SVC_RSC_POOL_GET_QUESTION_TEMPLATES)
```

Comment:
why do the work manager and pool manager have overlapping APIs?

(code looks identical or at least very close to work mgr?)

Reply:
It is identical. I'll explain the goals/rational in person tomorrow. There might be better ways of it. 

FYI, in this comment and the previous one, changes from PR #425 are bleeding in.